doc/content/devices-and-gateways/bytes/_index.md

```diff
@@ -160,7 +168,7 @@ int myVal = ((int)(payload[0]) << 8)
 
 If the range of expected values is bigger than 65536 we can use the same trick. The only difference is that we have to manually shift bits when we encode on Arduino, just like we did in the payload function.
 
-Let's say we need to encode a [long](https://www.arduino.cc/en/Reference/Long) which uses 4 bytes for a range up to 4294967296.
+Let us say we need to encode a [long](https://www.arduino.cc/en/Reference/Long) which uses 4 bytes for a range up to 4294967296.
```

Comment:
Thanks for correcting these, but I don't think we need to keep nitpicking the language this way. Nobody would ever say "Let us" and I disagree in general that the contractions are a problem.